scripts/ci/gitlab/pipeline/build.yml

```
@@ -26,10 +26,56 @@
       --extra-dependencies "$EXTRA_DEPENDENCIES"
       --companion-overrides "$COMPANION_OVERRIDES"
 
+.check-runtime-migration:
+  extends: .check-dependent-project
+  variables:
+    DEPENDENT_REPO: polkadot
+    COMPANION_OVERRIDES: |
+      substrate: polkadot-v*
+      polkadot: release-v*
+    COMPANION_CHECK_COMMAND: >
+      time cargo build --release --locked -p "$NETWORK"-runtime &&
+      time cargo run --locked --release --features try-runtime try-runtime \
+          --runtime ./target/release/wbuild/"$NETWORK"-runtime/target/wasm32-unknown-unknown/release/"$NETWORK"_runtime.wasm \
+          --chain=${NETWORK}-dev \
+          on-runtime-upgrade --checks=pre-and-post live --uri wss://${NETWORK}-try-runtime-node.parity-chains.parity.io:443
+
 # Individual jobs are set up for each dependent project so that they can be ran in parallel.
 # Arguably we could generate a job for each companion in the PR's description using Gitlab's
 # parent-child pipelines but that's more complicated.
 
+check-runtime-migration-polkadot:
+  extends:
+    - .check-runtime-migration
+  variables:
+    NETWORK: polkadot
+  rules:
+    - if: $CI_COMMIT_REF_NAME =~ /^[0-9]+$/ #PRs
```

```suggestion
check-runtime-migration-polkadot:
  extends:
    - .check-runtime-migration
    - .test-refs-no-trigger-prs-only
  variables:
    NETWORK: polkadot
```

Please use `.test-refs-no-trigger-prs-only` for all `check-runtime-*` jobs

---

Yep, you're right, that also makes sense